components/cdc/src/txn_source.rs

```diff
@@ -0,0 +1,112 @@
+// Copyright 2023 TiKV Project Authors. Licensed under Apache-2.0.
+
+// TiCDC uses 1 - 255 to indicate the source of TiDB.
+// For now, 1 - 15 are reserved for TiCDC to implement BDR synchronization.
```

Comment:
Seems it's not `1~15` for BDR synchronization.

Reply:
For now, it is. It will probably be expanded in the future.

Reply:
How about saying "16-255 are reserved for future use"?

Reply:
Could you draw a bitmap in comments?

Reply:
> How about saying "16-255 are reserved for future use"?

I already say `// 16 - 255 are reserved for extendability.`

Reply:
> Could you draw a bitmap in comments?

I tried it. But the max length of comments is 80, so I can not really draw a bitmap in comments.

Reply:
Added a simple one: https://github.com/tikv/tikv/pull/14629/commits/2d227adc68bb4e5504587552ff157a0e3c4c4aa7